spec/Ranges.tex

```diff
@@ -475,8 +475,10 @@ \subsection{Range Assignment}
 \item An implicit conversion is allowed from \chpl{idxType} of the source range
        to \chpl{idxType} of the destination range type,
 \item the two range types have the same \chpl{boundedType}, and
-\item either the destination range is stridable or the source range's
-      stride is 1.
+\item either the destination range is stridable or the source range
+      is not stridable.
+\item Or an explicit conversion is used to make the source range match the
+      type of the destination range.
```

Comment:
This section is a bit confusing because I think there's a missing "if". Here's what sounds right in my head:

An implicit converstion is allowed from {idxType} of the source range to {idxType} of the destination range type if the two range types have the same {boundedType} and either the destination range is stridable, the source range is not stridable, or an explicit conversion is used to make the source range match the type of the destination range.